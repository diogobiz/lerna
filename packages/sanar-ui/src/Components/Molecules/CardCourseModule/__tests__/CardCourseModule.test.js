import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCardCourseModule from '../CardCourseModule'

it('renders correctly', () => {
    const component = (
        <ESCardCourseModule>
            <h1>ESCardCourseModule</h1>
        </ESCardCourseModule>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
