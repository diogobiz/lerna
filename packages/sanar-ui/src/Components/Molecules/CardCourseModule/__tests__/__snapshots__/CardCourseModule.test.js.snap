// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-card es-card es-car-course-module ant-card-bordered ant-card-small"
  data-testid="es-card-course-module"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="es-car-course-module__content"
    >
      <div
        className="es-car-course-module__content__module-data"
      >
        <div
          className="ant-skeleton es-skeleton es-skeleton--dark"
        >
          <div
            className="ant-skeleton-content"
          >
            <ul
              className="ant-skeleton-paragraph"
            >
              <li
                style={
                  Object {
                    "width": undefined,
                  }
                }
              />
              <li
                style={
                  Object {
                    "width": "61%",
                  }
                }
              />
            </ul>
          </div>
        </div>
      </div>
      <div
        className="es-car-course-module__content__image-container mb-sm"
      >
        <div
          className="es-car-course-module__content__image-container--image"
          style={
            Object {
              "backgroundImage": "url(undefined)",
            }
          }
        />
      </div>
    </div>
    <div
      className="ant-row-flex ant-row-flex-end ant-row-flex-middle es-row es-car-course-module__footer"
      style={Object {}}
    >
      <div
        className="ant-col es-col"
        style={Object {}}
      >
        <button
          className="es-button pr-no es-button__xsmall es-button__variant--text bold ant-btn"
          data-testid="es-button"
          type="button"
        >
          <span>
            Ver aulas
          </span>
          <i
            className="anticon es-eva-icon es-eva-icon--large"
          >
            <i
              data-eva="chevron-right-outline"
              data-eva-fill="#fff"
              data-eva-height="18px"
              data-eva-width="18px"
            />
          </i>
        </button>
      </div>
    </div>
  </div>
</div>
`;
